release-notes/release-notes5103.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: TinyMCE 5.10.3
+title_nav: TinyMCE 5.10.3
+description: Release notes for TinyMCE 5.10.3
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+{{site.productname}} 5.10.3 was released for {{site.enterpriseversion}} and {{site.cloudname}} Tuesday, February 15<sup>th</sup>, 2022. It includes {{site.productname}} 5.10.3 and additional changes to premium plugins. These release notes provide an overview of the changes for {{site.productname}} 5.10.3, including:
+
+- [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges)
+- [General bug fixes](#generalbugfixes)
+- [Security fixes](#securityfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium self-hosted server-side component changes
+
+The {{site.productname}} 5.10.3 release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
```

Comment:
Suggested tweak:

\+ The {{site.productname}} 5.10.3 release includes changes affecting the {{site.productname}} **self-hosted** services for the following plugins:

Reply:
This is a template thing, so if we're changing this we need to change others: https://github.com/tinymce/tinymce-docs/blob/main/_new_content_templates/blank_releasenotes.md#accompanying-premium-self-hosted-server-side-component-changes